src/main/kotlin/lotto/model/LottoResult.kt

```diff
@@ -0,0 +1,30 @@
+package lotto.model
+
+const val MINIMUM_MATCHES = 3
```

Comment:
How about moving `MINIMUM_MATCHES` into the companion object of `LottoResult` 

https://kotlinlang.org/docs/object-declarations.html#companion-objects

Note: class members can access private members of their corresponding companion object. So `MINIMUM_MATCHES` does not have to be public in this case. 🙂